admin/scss/bootstrap/_variables.scss

```diff
@@ -0,0 +1,661 @@
+// Variables
+
+// Table of Contents
+//
+// Colors
+// Options
+// Spacing
+// Body
+// Links
+// Grid breakpoints
+// Grid containers
+// Grid columns
+// Fonts
+// Components
+
+// General variable structure
+//
+// Variable format should follow the `$component-modifier-state-property` order.
+
+// Colors
+//
+// Grayscale and brand colors for use across Bootstrap.
+
+$gray-dark:                 #373a3c !default;
+$gray:                      #55595c !default;
```

Comment:
Do we really want to name a variable `$gray`.

Also, can't we use lighten / darken to adjust the colours?

I'm not a fan of naming variables after colours because if we decide that grey should now be very dark blue, then we either have a variable called `$gray` that is now returning blue colours OR we have to rename the variable (in which case we might as well do a search and replace on the colour hex anyway.

I've seen this kind of variable called `$scaffolding` (which makes sense if it's used for backgrounds, borders, etc.) or maybe `$base` or `$neutral` to better describe it